### PR TITLE
[skip ci]retry 3 times for build

### PR DESCRIPTION
**Pull request by PengTian0**
Sometimes the stage build will fail because of poor network.
This PR will retry 3 times for package build and images build.

### CODE REVIEW

**Comment by PengTian0** [comment]
@panpan0000 @changev @yyscamper @anhou

File: jobs/shareMethod.groovy [hunk 1]
```diff
@@ -41,7 +41,9 @@ def getLockedResourceName(resources,label_name){
 
 def buildAndPublish(){
     stage("Packages Build"){
-        load("jobs/build_debian/build_debian.groovy")
+        retry(3){
```
Comment (anhou) [inline jobs/shareMethod.groovy]: 1. 3 exist in multiple places, please assigned '3' to a variable name to make it more meaningful and easy to change
2. also please add a annotation about why 3, or just 'it's a experience value'

**Review by anhou — APPROVED** [review]
👍 with comments updated

File: jobs/shareMethod.groovy (hunk 1 above)
Comment (PengTian0) [reply jobs/shareMethod.groovy]: Thanks for your review!
The item 1 is fixed.
About item2:
3 is just a experience value

**Comment by changev** [comment]
:+1:

**Comment by PengTian0** [comment]
@changev Any failure in the 
'''
load("jobs/build_docker/build_docker.groovy")
'''

**Pull request merged**